### PR TITLE
Rate limiting improvement

### PR DESCRIPTION
**Pull request by dpotkoc**

### CODE REVIEW

File: .github/workflows/code_quality.yml [hunk 1]
```diff
@@ -89,12 +89,6 @@ jobs:
             php: "8.2"
             composer: "v2"
             opensearch: "2.5.0"
-
-          - magento: "2.4.6"
```
Comment (fredden) [inline .github/workflows/code_quality.yml:93]: Removing this looks out of scope for a rate limiting change.

Comment (vvuksan) [reply .github/workflows/code_quality.yml:93]: The test was added in error so will just leave the removal here.
